src/app/wallet/send/send.service.ts

```diff
@@ -173,7 +173,7 @@ export class SendService {
   getSendParams(
     anon: boolean, address: string, amount: number, comment: string,
     narration: string, ringsize: number, numsignatures: number) {
-    const params: Array<any> = [address, amount, '', '', false];
+    const params: Array<any> = [address, amount, '', '', true];
```

Comment:
Including the fee by default should _not_ be default behaviour.

Reply:
Okay, So I guess we should use a checkbox for select & unselect `Send al`l amount ..? 

if someone select >> Send all checkbox then amount field will be disable and if use unchecked `Send All` amount field will empty.  (at the same time SendService will also update true/false)